src/mono/mono/mini/mini-s390x.h

```diff
@@ -149,7 +151,7 @@ struct SeqPointInfo {
 /*-----------------------------------------------*/
 
 #define MONO_MAX_XREGS			31
-#define MONO_ARCH_CALLEE_XREGS		0x0
+#define MONO_ARCH_CALLEE_XREGS 0xFFFEFFFE
```

Comment:
If we manage to eliminate any hard-coded vr16 use (I think we can), it wouldn't need to be reserved here.